src/Acuminator/Acuminator.Vsix/VSPackage.resx

```diff
@@ -118,10 +118,13 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="110" xml:space="preserve">
-    <value>Acuminator Code Analysis and Colorizer Tool</value>
+    <value>Acuminator Code Analysis and Developer Productivity Tool</value>
```

Comment:
@EAndrosova please review the changes in the extension description displayed in the Help -> About section